src/app/cdk/side-bar/modals/modal-email/modal-email.component.html

```diff
@@ -1,100 +1,302 @@
-            ></app-privacy-selector>
+          <div class="col actions-wrapper no-gutters">
+            <div class="actions-sub-wrapper"></div>
+            <div class="actions-sub-wrapper">
+              <app-privacy-selector
+                formControlName="visibility"
+              ></app-privacy-selector>
+              <div></div>
+            </div>
           </div>
         </div>
+        <app-alert-message
+          *ngIf="email.primary && !showEmailAsVerified(email.putCode)"
+          class="orc-font-body-small"
+        >
+          <strong class="orc-font-body" i18n="@@side-bar.pleaseVerify"
+            >Please verify your primary email address!</strong
+          >
+
+          <p>
+            <ng-container i18n="@@side-bar.youNeedToVerify">
+              You need to verify your primary email in order to access all of
+              ORCID’s editing features.
+            </ng-container>
+            <br />
+            <ng-container i18n="@@side-bar.youNeedToVerify2">
+              To verify your email, click the link in the message sent to:
+            </ng-container>
+          </p>
+          <p class="email-container">{{ email.value }}</p>
+
+          <p>
+            <strong i18n="@@side-bar.needHelp"> Need help?</strong> <br />
+            <ng-container i18n="@@side-bar.visitOur"> Visit our </ng-container>
+            <a
+              href="https://support.orcid.org/hc/articles/360006973793"
+              i18n="@@side-bar.knowledge"
+              >knowledge base
+            </a>
+            <ng-container i18n="@@side-bar.orContact">
+              or contact
+            </ng-container>
+
+            <a
+              i18n="@@side-bar.support"
+              href="https://support.orcid.org/hc/en-us/requests/new"
+            >
+              support.
+            </a>
+          </p>
+        </app-alert-message>
       </ng-container>
     </div>
 
     <div class="row">
-      <h3 class="col">Other email address</h3>
+      <h3 class="col" i18n="@@side-bar.otherEmails">Other email address</h3>
     </div>
 
+    <!-- ADDITIONAL EMAILS -->
+
     <ng-container *ngFor="let email of emails">
       <div
         *ngIf="!email.primary"
-        class="row email-input-line no-wrap"
-        [formGroupName]="email.value"
+        class="row email-input-line"
+        [formGroupName]="email.putCode"
       >
-        <div class="col l12 m8 s4">
+        <div class="col l12 m8 s4 email-input-container">
           <mat-form-field
             appearance="outline"
-            class="mat-form-field-min"
-            [ngClass]="{ 'email-activated': true }"
+            class="mat-form-field-min verify-icon"
+            [ngClass]="{
+              'email-activated': showEmailAsVerified(email.putCode)
+            }"
           >
-            <input #emailInput matInput formControlName="email" />
+            <input
+              [id]="email.putCode"
+              matInput
+              formControlName="email"
```

Comment:
The placeholder is missing here

Reply:
same as https://github.com/ORCID/orcid-angular/pull/692#discussion_r581049148